README.md

```diff
@@ -227,21 +227,24 @@ For example: `export MINIO_SERVER_URL="https://minio.example.net"`
 `mc` provides a modern alternative to UNIX commands like ls, cat, cp, mirror, diff etc. It supports filesystems and Amazon S3 compatible cloud storage services. Follow the MinIO Client [Quickstart Guide](https://docs.min.io/docs/minio-client-quickstart-guide) for further instructions.
 
 # Upgrading MinIO
-MinIO server supports rolling upgrades, i.e. you can update one MinIO instance at a time in a distributed cluster. This allows upgrades with no downtime. Upgrades can be done manually by replacing the binary with the latest release and restarting all servers in a rolling fashion. However, we recommend all our users to use [`mc admin update`](https://docs.min.io/docs/minio-admin-complete-guide.html#update) from the client. This will update all the nodes in the cluster simultaneously and restart them, as shown in the following command from the MinIO client (mc):
+Upgrades require zero downtime in MinIO, all upgrades are non-disruptive, all transactions on MinIO are atomic. So upgrading all the servers simultaneously is the recommended way to upgrade MinIO.
```

Comment:
Zero downtime implies the server never goes down. The server must go down for a measurable amount of time, during which applications typically re-try writes. The *application* itself though should see no downtime, just a brief blip of increased 404s. We should say that to avoid users expecting something unreasonable.

```suggestion

MinIO requires upgrading all servers simultaneously with a single deployment-wide restart. Since transactions in MinIO are atomic *and* applications typically retry failed S3 API operations by default, upgrades are minimally-disruptive and do not result in application downtime or lost data.
```

Reply:
> Zero downtime implies the server never goes down. The server must go down for a measurable amount of time, during which applications typically re-try writes. The *application* itself though should see no downtime, just a brief blip of increased 404s. We should say that to avoid users expecting something unreasonable.
> 
> ```suggestion
> 
> MinIO requires upgrading all servers simultaneously with a single deployment-wide restart. Since transactions in MinIO are atomic *and* applications typically retry failed S3 API operations by default, upgrades are minimally-disruptive and do not result in application downtime or lost data.
> ```

No we are going to say Zero downtime, from application point of view a retry is a zero downtime. The transactions are retried and there is no state loss anywhere.